### PR TITLE
add DatalandEDC and PostgreSQL to architecture diagram

### PR DESCRIPTION
**Pull request by ppuetsch**
# Pull Request Architecture Diagram including DatalandEDC
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [ ] The code has been manually inspected
- [ ] The new feature was observed "in running software"
- [ ] The DoD was checked for completeness
- [ ] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one E2E Test exists testing the new feature
- [ ] Documentation is updated as required
- [ ] The new version is deployed to preview using this branch
  - [ ] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [ ] The new feature is manually used/tested/observed on preview

### CODE REVIEW

File: documentation/architecture.drawio [hunk 1]
```diff
@@ -1,20 +1,20 @@
-<mxfile host="app.diagrams.net" modified="2022-03-03T06:58:21.012Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36" etag="jawmesXoiLuioSDJKZNp" version="16.6.4" type="github">
+<mxfile host="app.diagrams.net" modified="2022-03-31T09:36:37.073Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36" etag="lK_5tippkE5IevXrBJ4l" version="17.1.3" type="github">
```
Comment (StephanSinn) [inline documentation/architecture.drawio:1]: Blaue Box. Ist ein "Reverse Proxy" nicht schon der Begriff für Inbound Kommunikation? 
Also: "Dataland Inbound Reverse Proxy" should be "Dataland Reverse Proxy" OR "Dataland Inbound Proxy"

File: documentation/architecture.drawio [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
   <diagram id="EgJ7EEfOsz50NrqOtSAN" name="Page-1">
```
Comment (StephanSinn) [inline documentation/architecture.drawio:2]: Der Skyminder Client hat eine extra Box (hellblau innerhalb Dataland Backend), EDC Clients nicht. Das sollte konsistent sein.

File: documentation/architecture.drawio [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-    <mxGraphModel dx="1022" dy="474" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="1169" pageHeight="827" math="0" shadow="0">
+    <mxGraphModel dx="1955" dy="1185" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="1169" pageHeight="827" math="0" shadow="0">
```
Comment (StephanSinn) [inline documentation/architecture.drawio:3]: Mir fällt es schwer aus dem Architecture Diagram hilfreiche architektur schemata abzuleiten, zB. was ist ein MicroService

**Review by StephanSinn — CHANGES_REQUESTED** [review]

File: documentation/architecture.drawio (hunk 3 above)
Comment (ppuetsch) [reply documentation/architecture.drawio]: wie besprochen: PostgreSQL bleibt gemäß Abstimmung Teil des Backend, ansonsten noch Legendeneintrag für "Datalandinternal Microservice" hinzugefügt.

File: documentation/architecture.drawio (hunk 2 above)
Comment (ppuetsch) [reply documentation/architecture.drawio:2]: Habe EDC Client hinzugefügt

File: documentation/architecture.drawio (hunk 1 above)
Comment (StephanSinn) [reply documentation/architecture.drawio]: Oki doki, danke für den Link.

File: documentation/architecture.drawio (hunk 2 above)
Comment (StephanSinn) [reply documentation/architecture.drawio:2]: lgtm

File: documentation/architecture.drawio (hunk 3 above)
Comment (StephanSinn) [reply documentation/architecture.drawio]: lgtm

**Pull request merged**